sched/definitions.go

```diff
@@ -17,3 +17,33 @@ type Task struct {
 	Command    []string // Argv to run
 	SnapshotId string
 }
+
+type JobState int
+
+const (
+	// An unambiguous 0-value.
+	UNKNOWN JobState = iota
```

Comment:
So in the API design as I understand it,

If you have a JobId it means your Job was successfully scheduled, since we return this as part of accepting the work.  So if they have a JobId you know your job was scheduled successfully.   Why do we need this extra status?